src/app/contextBroker/contextBroker.cpp

```diff
@@ -204,6 +205,14 @@ using namespace orion;
 
 /* ****************************************************************************
 *
+* Global vars
+*/
+static bool isFather = false;
```

Comment:
isFather is a bit generic... maybe isProcessFather (or something like that) would be more clear.